modules/openapi-generator/src/main/resources/python-fastapi/model.mustache

```diff
@@ -1,9 +1,10 @@
 # coding: utf-8
 
+from __future__ import annotations
```

Comment:
Since this change is supported by >=3.7, the version number needs to be changed in all places: readme.mustache, docker-compose.mustache, etc. thx

Reply:
Sure, can do.

However, I found that my fix is not sufficient. Pydantic cannot deal with the forward references (and breaks in, e.g., the generation of the model's JSON schema) _unless_ `BaseModel.update_forward_refs(cls)` is called on the respective model class.

I will update the PR accordingly, but I probably won't get around to it today.

Reply:
I changed the minimum Python version in {Dockerfile,README,setup_cfg}.mustache and added a call to `update_forward_refs()` on all model classes.

The FastAPI service I built with this generator (incl. a rudimentary yet functional implementation) both responds as expected to the REST requests and serves the API documentation.